testsetup/testsetup.go

```diff
@@ -24,6 +24,7 @@ const (
 	EnvLoadTestAPI               = "GOVIM_LOAD_TEST_API"
 	EnvDisableIncrementalSync    = "GOVIM_DISABLE_INCREMENTALSYNC"
 	EnvDisablePopupWindowBalloon = "GOVIM_DISABLE_POPUPWINDOWBALLOON"
+	EnvDisableSignPlace          = "GOVIM_DISABLE_SIGNPLACE"
```

Comment:
👍

Reply:
Just to clarify, I'm giving this a 👍 because this is an immature new feature that we want to test over time with the setting of this env variable being a nuclear option in case people find issues.